test/Twig/Tests/EnvironmentTest.php

```diff
@@ -54,11 +54,11 @@ public function testGlobals()
         $globals = $twig->getGlobals();
         $this->assertEquals('bar', $globals['foo']);
 
-        // globals can be modified after runtime init
+        // globals can be modified after a template has been loaded
         $twig = new Twig_Environment($this->getMock('Twig_LoaderInterface'));
         $twig->addGlobal('foo', 'foo');
         $twig->getGlobals();
-        $twig->initRuntime();
+        $twig->loadTemplate('index');
```

Comment:
This is more semantically correct. Calling `initRuntime` is just a side-effect of loading a template.